scripts/experimental/test_solar_uav_mission.py

```diff
@@ -237,10 +238,11 @@ def define_vehicle():
     net.avionics        = avionics      
 
     # Component 8 the Battery # I already assume 250 Wh/kg for batteries
-    bat = SUAVE.Components.Energy.Storages.Battery()
+    bat = SUAVE.Components.Energy.Storages.Constant_Mass.Battery_Lithium_Ion()

```

Comment:
was looking for ```Storages.Batteries.Constant_Mass.Lithium_Ion```, to avoid the battery in the final name.  let's discuss here if this doesn't seem right